redirects.yaml

```diff
@@ -300,3 +300,9 @@ redirects:
   # Design system
   - source: /design-system/component/action
     destination: /design-system/component/fab
+
+  # Redirect for Attribution Reporting API content (ex-conversion measurement) moved to dcc
+  - source: /conversion-measurement/
```

Comment:
Let's keep it together with redirects in line 288 (Redirects to https://developer.chrome.com/.)

Reply:
@maudnals PR is approved, so once you move the code a bit further up as Ewa suggested you can merge.

Reply:
Moved, I'll merge once the build succeeds